Rpackage/R/mmit.cv.R

```diff
@@ -99,6 +91,11 @@ mmit.cv <- structure(function(target.mat, feature.mat,
   param_grid$min_sample <- c(2, 5, 10, 20)
   param_grid$loss <- c("hinge", "square")
   
-  result <- mmit.cv(target.mat, feature.mat, param_grid, scorer = mse, n_cpu = 4)
-  
+  if(require(parallel)){
+    cl <- makeCluster(detectCores())
+    registerDoParallel(cl)
+  }
+  if(require(future)){ plan(multiprocess)}
+  result <- mmit.cv(target.mat, feature.mat, param_grid, scorer = mse)
+  if(require(parallel)){ stopCluster(cl)}
```

Comment:
please delete

Reply:
actually all of the ex=function(){} blocks were example code that was converted to Rd examples by inlinedocs. but since you are using roxygen2 now, it does not make sense to include these anymore. can you please either (1) delete them, or (2) move them into the roxygen2 examples?

Reply:
ok...i have already given roxygen2 examples so I guess i'll remove these ones....I was using them for debugging purpose...but in final package they shouldnt be there i guess

Reply:
done